docs/tutorials/state_histograms.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p>Recommend "For example" rather than "For eg"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done.